metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -72,6 +72,13 @@ def ingest() -> None:
     default=False,
     help="If enabled, ingestion runs with warnings will yield a non-zero error code",
 )
+@click.option(
+    "--safe",
```

Comment:
Seems like safe doesn't match what it is actually doing. 

Can we rename this to --suppress-error-logs ? to be more explicit about what you are doing here.

Reply:
Thanks for the feedback! Done!